bin/openstack-install

```diff
@@ -110,8 +114,10 @@ def populate_config(opts):
         _cfg = yaml.load(utils.slurp(presaved_config))
     else:
         _cfg = {}
-    # Update cfg dict with command line opts
-    _cfg.update(vars(opts))
+    # Update cfg dict with command line opts unless
+    # we're in headless mode
```

Comment:
yea i need to rework this logic, the trick is to not set options in the config to null if they are set but not defined on the cli..

im going to rework it to just merge left and not unset valid config items